src/vscode-cosmosdb.api.d.ts

```diff
@@ -59,7 +65,7 @@ export interface DatabaseTreeItem extends DatabaseAccountTreeItem {
 
 export type CosmosDBResourceType = 'DatabaseAccount' | 'Database';
 
-export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table';
+export type CosmosDBApiType = 'Mongo' | 'SQL' | 'Graph' | 'Table' | 'Postgres';
```

Comment:
So all of the naming in this file is still Cosmos DB-specific. Are we sure that it will work as-is for PostgreSQL? For example, Postgres is not a "_Cosmos DB_ API Type".

Reply:
The checked the properties mentioned in this api file are very specifically created for the api, don't have direct connection to the rest of the Databases code.

Reply:
Well I think this release is the right time to change the naming. It was accurate before "Postgres" was added - and now it's not. Better to clump the naming changes and functionality changes together when it comes to an API